website/static/js/citationList.js

```diff
@@ -14,8 +14,19 @@ var STYLES = {
 
 var ctx = window.contextVars;
 
+var escapeHTML = function(s) {
```

Comment:
This function can be pulled out to be a helper function in osfhelper.js. It has a duplicate below